workspaces/lightspeed/plugins/lightspeed/src/hooks/useConversationMessages.ts

```diff
@@ -313,12 +313,11 @@ export const useConversationMessages = (
 
                   if (documents.length) {
                     lastMessage.sources = {
-                      sources: documents.map(
-                        (doc: { doc_title: string; doc_url: string }) => ({
-                          title: doc.doc_title,
-                          link: doc.doc_url,
-                        }),
-                      ),
+                      sources: documents.map((doc: ReferencedDocument) => ({
+                        title: doc.doc_title,
+                        link: doc.doc_url,
+                        body: doc.doc_description,
```

Comment:
FYI, RCS uses `text` but it actually returns the entire doc text, so we may need to get a substring of this value for doc_description 🤔 

```class RagChunk:
    """Model representing a RAG chunk.

    Attributes:
        text: The text used as a RAG chunk.
        doc_url: The URL of the doc from which the RAG chunk comes from.
        doc_title: The title of the doc.
    """

    text: str
    doc_url: str
    doc_title: str
```

Reply:
I am okay with getting a shorter text of the document or `subtitle` field from `master.adoc`.